### PR TITLE
Switch to GitHub style CODEOWNERS.

### PR DESCRIPTION
**Pull request by ddunbar**
- See https://help.github.com/articles/about-codeowners/

### CODE REVIEW

**Comment by aciidb0mb3r** [comment]
I've been thinking to do this for a while 👍

File: CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+# (W), PGP key ID and fingerprint (P), description (D), and snail-mail address
+# (S).
+
+# N: Daniel Dunbar
+# E: daniel_dunbar@apple.com
+# D: Everything in llbuild not covered by someone else
+
+###
+
+# The following lines are used by GitHub to automatically recommend reviers.
```
[the remark this answers is not in the document]
Comment (ddunbar) [reply CODEOWNERS]: Fixed

**Comment by dmbryson** [comment]
@swift-ci please smoke test

**Comment by dmbryson** [comment]
@swift-ci please smoke test

**Pull request merged**